balancer-labs/frontend-v2
# veBAL gauges table + voting

# Description

Add veBAL gauges table with overall weights and user voting weight. 
Add ability to vote on gauges. 

Not working yet, most code is implemented but it still needs work to make it all work. 

## Type of change


## How should this be tested?

- Visit `/vebal`
- Stake BAL and ensure your staked BAL is shown at the top of the `/vebal` page.
- Vote on a gauge with any vote level between 1 - 10,000.
- Ensure gauge vote levels and and your user votes are shown correctly in the table. 

## Visual Context

https://www.loom.com/share/e796c23643854f45bad73896fe495130

## Checklist:
